spec/jobs/notifications/welcome_notification_job_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe Notifications::WelcomeNotificationJob, type: :job do
+  describe "::perform_now" do
+    let(:user) { double }
+    let(:welcome_notification_service) { Notifications::WelcomeNotification::Send }
```

Comment:
What do you think about using `double` as a  `welcome_notification_service` + pass it to the `perform_now`?
It's more explicit in my opinion.

Reply:
I have changed that spec accordingly.